en/includes/apis/scim2/build-scim2-user-creation-payload.md

```diff
@@ -0,0 +1,244 @@
+
+!!! note
+
+    - You may find these schemas on the {{product_name}} Console by navigating to **User Attributes & Stores** > **Attributes** > **SCIM 2.0**. Learn how to [Manage SCIM 2.0 attribute mappings]({{base_path}}/guides/users/attributes/manage-scim2-attribute-mappings).
+
+For a user attribute,
+
+- if it is mapped to the **Core Schema** or the **User Schema**, the schema URI does not need to be included in the SCIM payload.
+
+    ```json
+    {
+      "name": {
+        "givenName": "Kim",
+        "familyName": "Berry"
+      },
+      "username": "kimberry"  
+    }
+    ```
+
+- If it is mapped to **Enterprise Schema** or **Custom Schema**, it needs to be placed under the namespace of the corresponding schema.
+
+
+    ```json
+    {
+     "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User": {
+       "employeeNumber": "1234A"
+     },
+     "urn:scim:wso2:schema": {
+       "customAttribute": "xyz"
+     }
+    }
+    ```
+
+## Step 2 : Identify the attribute type
+
+Each SCIM attribute belongs to one of the following types, which determine how the attribute is formatted in the payload.
+
+
+- **Single-valued Attributes** contain a single value.
+
+    - **Simple Attributes** contain a single attribute.
+
+          ```json
+          {
+            "userName": "kim"
+          }
+          ```
+
+
+    - **Complex Attributes** contain multiple sub-attributes.
+
+        ```json
+        {
+          "name": {
+            "givenName": "Kim",
+            "familyName": "Berry"
+          }
+        }
+        ```
+
+- **Multi-Valued Attributes** hold multiple values
+
+      - **Simple Attributes** contain a single attribute.
+
+        ```json
+        {
+            "devices": ["d1", "d2"]
+        }
+        ```
+
+      - **Complex Attributes** contain multiple sub-attributes.
+
+        ```json
+        {
+          "emails": [
+            {
+              "value": "kim@gmail.com",
+              "primary": true
+            },
+            {
+              "type": "work",
+              "value": "kim@wso2.com"
+            }
+          ]
+        }
+        ```
+
+!!! info
+    The following references provide comprehensive information about SCIM attribute types and their respective definitions. These details can help identify the type of attributes used in SCIM 2.0 payloads:
+
+      - For attributes under core schema, user schema and SCIM2 specification-defined enterprise schema, refer to [RFC 7643 Section 8.7.1](https://datatracker.ietf.org/doc/html/rfc7643#section-8.7.1).
+
+      {% if product_name == "WSO2 Identity Server" %}
+      - For {{product_name}}-defined enterprise schema attributes, refer to the `scim2-schema-extension.config` file located in the `<IS_HOME>/repository/conf/` directory.
+      {% endif %} 
+
+      - For custom schema attributes, check the `type` meta attribute of the mapped local attribute.
```

Comment:
```suggestion
      - For custom schema attributes, check the `dataType` meta attribute of the mapped local attribute.
```

Reply:
https://is.docs.wso2.com/en/6.1.0/references/extend/provisioning/add-scim2-custom-user-schema-support/#add-custom-schema-dialect:~:text=Additional%20Properties%20element.-,dataType,-multiValued